test/scripts/common.inc.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# Include file with common parts shared for local and ephemeral
+#
+
+function error {
+    local err=$?
+    printf "ERROR: %s\n" "$*" >&2
+    exit $err
+}
+
+ORG_ID="${ORG_ID:-12345}"
+
+IDENTITY_USER="$( ./tools/bin/xrhidgen -org-id "${ORG_ID}" user -is-active=true -is-org-admin=true -user-id test -username test | base64 -w0 )"
+[ "${IDENTITY_USER}" != "" ] || error "IDENTITY_USER got empty: check that you executed 'make install-tools'"
```

Comment:
And pipes strike again! All scripts should set `set -eo pipefail` (exit on uncatched error, catch pipe failures).

In this case, it would be better to add an explicit check for presence of the binary:

```
SRCDIR="$(dirname "${BASH_SOURCE[0]}")"
BASEDIR="$(dirname $(dirname $SRCDIR))"
XRHIDGEN="${BASEDIR}/tools/bin/xrhidgen"

if [[ ! -x $XRHIDGEN ]]; then
    error "${XRHIDGEN} is missing, run 'make install-tools'"
    exit 2
fi
```